main/src/main/java/org/mobilitydata/gtfsvalidator/runner/ValidationRunner.java

```diff
@@ -52,6 +52,7 @@ public class ValidationRunner {
 
   private static final FluentLogger logger = FluentLogger.forEnclosingClass();
   private static final String GTFS_ZIP_FILENAME = "gtfs.zip";
+  private static NoticeContainer noticeContainer;
```

Comment:
Does this have to be static? 
The way it is now we can't instantiate more than 1 ValidationRunner.
Even if we don't do it now, it's a simple fix that might prevent errors in the future.

Reply:
I think so. The createFromPath method in the GtfsInput class is static. If notice container is not static, an error "Non-static field 'noticeContainer' cannot be referenced from a static context" would occur. I keep it as is for now.